lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -62,6 +64,7 @@
 	private final RepositorySystemSession repoSession;
 	private final MavenSession mavenSession;
 	private final BuildPluginManager buildPluginManager;
+	private static Properties environmentProperties;
```

Comment:
I'm not so sure about making this static

Reply:
I think it's OK.